geode-assembly/src/upgradeTest/java/org/apache/geode/session/tests/TomcatSessionBackwardsCompatibilityTestBase.java

```diff
@@ -105,7 +108,10 @@ protected void startServer(String name, String classPath, int locatorPort) throw
   }
 
   protected void startLocator(String name, String classPath, int port) throws Exception {
-    locatorDir = tempFolder.newFolder("locator").getPath();
+    File locatorFile = new File("locator_dir_" + this.getClass().getSimpleName() + "_"
+        + testName.getMethodName().replace("[", "").replace("]", ""));
```

Comment:
Isn't it possible for the test name to include invalid filesystem characters inside the `[]` part? It looks like the only params might be the version number right now but if someone adds another param to test this could change. Perhaps we should just hash the name or something?